apps/web/src/pages/brand/tabs/BrandingForm.tsx

```diff
@@ -51,26 +57,17 @@ export function BrandingForm() {
         setValue('fontFamily', organization?.branding?.fontFamily);
       }
     }
-  }, [organization]);
+  }, [organization, setValue]);
 
-  function beforeUpload(files: File[]) {
-    setFile(files[0]);
-  }
-
-  useEffect(() => {
-    if (file) {
-      handleUpload();
-    }
-  }, [file]);
```

Comment:
removed this use effect as it's not needed the action of image upload will be fired on update click